public/app/features/plugins/built_in_plugins.ts

```diff
@@ -27,8 +27,8 @@ const prometheusPlugin = async () =>
   await import(/* webpackChunkName: "prometheusPlugin" */ 'app/plugins/datasource/prometheus/module');
 const mssqlPlugin = async () =>
   await import(/* webpackChunkName: "mssqlPlugin" */ 'app/plugins/datasource/mssql/module');
-const testDataDSPlugin = async () =>
-  await import(/* webpackChunkName: "testDataDSPlugin" */ '@grafana-plugins/grafana-testdata-datasource/module');
+// const testDataDSPlugin = async () =>
+//   await import(/* webpackChunkName: "testDataDSPlugin" */ '@grafana-plugins/grafana-testdata-datasource/module');
```

Comment:
This is just to demonstrate that this code is not being executed if the feature flag is enabled.

Reply:
Took it for a spin and it's working well. 👍